StudentsManager.Spa/src/index.css

```
@@ -12747,4 +12747,386 @@ details.soge-result-card summary.soge-result-history-toggle:focus-visible {
+.chatbot-message {
+    display: flex;
+    gap: 10px;
+    align-items: flex-start;
+    animation: chatMsgIn 0.25s ease both;
+}
+
+.chatbot-message--user {
+    flex-direction: row-reverse;
+}
+
+@keyframes chatMsgIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "chatMsgIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)